src/test/csfle.rs

```diff
@@ -310,3 +329,39 @@ async fn fle2v2_ok(name: &str) -> bool {
     }
     true
 }
+
+pub(crate) fn fill_kms_placeholders(
```

Comment:
This makes more sense here than in test_runner.rs; the only logic change is the conditional for temp creds for AWS.